library/cipher.c

```diff
@@ -298,7 +298,7 @@ int mbedtls_cipher_setkey( mbedtls_cipher_context_t *ctx,
         psa_status_t status;
         psa_key_type_t key_type;
         psa_key_usage_t key_usage;
-        psa_key_policy_t key_policy;
+        psa_key_attributes_t key_attributes;
```

Comment:
In commit "Update psa_import_key to PSA 1.0", should we be consistent in naming this between `cipher.c`, `pk.c`, `ssl_tls.c`, `pk_wrap.c`, etc.? Sometimes we use `key_attributes` and other times `attributes`.